src/fileshelter/ui/ShareDownload.cpp

```diff
@@ -53,8 +54,8 @@ ShareDownload::handlePathChanged()
 
 	if (!wApp->internalPathMatches("/share-download"))
 		return;
-
-	try
+    
```

Comment:
useless change (this makes history harder to follow)

Reply:
AH - I use convert tabs to spaces in the editor - you leave tabs - that's why it looked the same....